UI/frontend-plugins/decklink-output-ui/DecklinkOutputUI.cpp

```diff
@@ -0,0 +1,69 @@
+#include "DecklinkOutputUI.h"
+#include <obs-module.h>
+#include <util/platform.h>
+#include "decklink-ui-main.h"
+
+DecklinkOutputUI::DecklinkOutputUI(QWidget *parent)
+		: QDialog(parent),
+		  ui(new Ui_Output) {
+	ui->setupUi(this);
+
+	propertiesView = nullptr;
+
+	connect( ui->startOutput, SIGNAL( released() ), this, SLOT(StartOutput()) );
+	connect( ui->stopOutput, SIGNAL( released() ), this, SLOT(StopOutput()) );
+}
+
+void DecklinkOutputUI::ShowHideDialog() {
+	SetupPropertiesView();
+
+	if (!isVisible()) {
+		setVisible(true);
+	} else {
+		setVisible(false);
+	}
+}
+
+void DecklinkOutputUI::SetupPropertiesView() {
+	if (propertiesView != nullptr) {
+		delete propertiesView;
+	}
+
+	obs_data_t *settings = obs_data_create();
+
+	OBSData data = load_settings();
+	if (data != nullptr) {
+		obs_data_apply(settings, data);
+	}
+
+	propertiesView = new OBSPropertiesView(settings,
+											  "decklink_output",
```

Comment:
Style:  This and the next two lines are tabbed in way too far.  Please fix that.